ibis/backends/tests/test_string.py

```diff
@@ -1012,9 +1012,21 @@ def test_parse_url(con, result_func, expected):
     assert result == expected
 
 
-def test_capitalize(con):
-    s = ibis.literal("aBc")
-    expected = "Abc"
+@pytest.mark.parametrize(
+    ("inp, expected"),
+    [
+        (None, None),
+        ("", ""),
+        ("Abc", "Abc"),
+        ("aBc", "Abc"),
+        ("aBc def", "Abc Def"),
+        ("aBc-def", "Abc-Def"),
+        ("aBc1def", "Abc1def"),
+        ("bob's burgers", "Bob's Burgers"),
```

Comment:
This test case seems redundant with the above IMO

```suggestion
```

Reply:
ah, this was from when I thought the intended behavior was like .title(). I agree, I removed this now.